charts/pipelines-operator/templates/subscription.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
```

Comment:
Looks simple, simple is good.

Is this all we need to provision? I'll defer to @jayfray12 Will this provide everything your Quarkus CI chart depends on?

Reply:
Great work.  Looks good to me.
I pulled the branch and used the chart to install the Operator and then used my chart to install and run the pipeline.  
Works great!